process_pub_database/get_data_cpdl.py

```diff
@@ -8,220 +8,222 @@
 import time
 import unicodedata
 
+from utils import *
 
 
-def progress(count, total, suffix=''):
+def find_composer(bs_file):
+    """
+    Function to find the name and URL of a composer from an HTML page.
```

Comment:
You don't need to say "Function to..." here, we know that these are functions. Just say what it does.

It's nice to include data about the arguments and return values in a standardised formatted structure. We prefer the google docstring format: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html (see Args: and Returns:)